### PR TITLE
Add `cidon.de` server

### PR DESCRIPTION
**Pull request by unordentlich**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/cidon/background.png
Comment (maaaathis) [inline minecraft_servers/cidon/background.png:1]: Compress this file

File: minecraft_servers/cidon/background@2x.png
Comment (maaaathis) [inline minecraft_servers/cidon/background@2x.png:1]: Compress this file

File: minecraft_servers/cidon/manifest.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "server_name": "cidon",
+  "nice_name": "Cidon",
+  "direct_ip": "cidon.de",
+  "server_wildcards": [
+    "%.cidon.de"
+  ],
```
Comment (maaaathis) [inline minecraft_servers/cidon/manifest.json:7]: Is it really a wildcard?
image

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/cidon/background@2x.png
Comment (Pfarospa) [reply minecraft_servers/cidon/background@2x.png:1]: qdf

File: minecraft_servers/cidon/manifest.json (hunk 1 above)
Comment (unordentlich) [reply minecraft_servers/cidon/manifest.json]: I've removed the wildcards with https://github.com/LabyMod/server-media/pull/983/commits/00fb77febfa95ba10bd34ade623c7c2c6079b8f3

File: minecraft_servers/cidon/background@2x.png
Comment (unordentlich) [reply minecraft_servers/cidon/background@2x.png:1]: Fixed with https://github.com/LabyMod/server-media/pull/983/commits/06d3310cb3c71181ef003b39ea61db8db5672672

File: minecraft_servers/cidon/background.png
Comment (unordentlich) [reply minecraft_servers/cidon/background.png:1]: Fixed with https://github.com/LabyMod/server-media/pull/983/commits/06d3310cb3c71181ef003b39ea61db8db5672672

**Review by maaaathis — APPROVED** [review]
looks good to me 👍

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**